src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir.ttc);
+}
+
+.big-input {
+  width: 100%;
+  padding-top: 24px;
+  padding-bottom: 24px;
+  margin-top: 20px;
+  border-radius: 4px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  font-weight: 300;
+  position: relative;
+  display: block;
+  line-height: 21.86px;
+  height: 70px;
+}
+
+.page-form {
+  width: 100%;
+}
+
+.small-input {
```

Comment:
create a common class for both forms and set all repetitive rules there, and use additional class only for different rules